docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -0,0 +1,101 @@
+[role="xpack"]
+[[task-manager-health-monitoring]]
+== Task Manager health monitoring
+
+++++
+<titleabbrev>health monitoring</titleabbrev>
+++++
+
+The {kib} Task Manager has an internal monitoring mechanism keeping track of a variety of metrics which can be consumed via both a health monitoring API, and the {kib} Server Log.
+
+The health monitoring API is designed to be fast, providing a reliable endpoint which can be monitored.
+Consuming this endpoint doesn't cause additional load, but rather it returns the latest health checks made by the system. This design enables consumption by external services at a regular cadance without the concern of additional load to the system.
```

Comment:
That's up to the customer 🤷 
Monitoring endpoints are designed to be consumed in order to verify the system is up and running.

Many of companies will be using Kibana and Alerting to monitor their services, but they won't want to rely on Kibana/Alerting to monitor itself (if it's down it can't tell you it's down), so they'll use another service, such as Nagios, to consume this endpoint and alert them when Kibana/Alerting is down.

Reply:
I've added the word "monitoring", so it's "external monitoring services"